handlers/handler-loadbalance/src/main/java/io/servicecomb/loadbalance/LoadbalanceHandler.java

```diff
@@ -350,4 +349,15 @@ private void loadFilter(String filter, LoadBalancer lb) {
       }
     }
   }
+  public boolean isNotChanged(String loadbalancePolicy, String loadbalanceStrategy) {
```

Comment:
public boolean isChanged(String newPolicy, String newStrategy){
  return policy or strategy changed;
}

policy is no relationship to strategy, but you mix them together.

Reply:
i remember someone had give you a goold example at last PR of this issue......

Reply:
ok,i will try. thank you